.travis.yml

```diff
@@ -3,11 +3,11 @@ dist: focal
 language: node_js
 
 node_js:
-  - 14.21.2
+  - 16.20.1
```

Comment:
We should probably decide which version we will use for all of the backend Node.js repos. For the repos I updated I migrated directly to `18.18.1` as any earlier versions are EOL (according to https://nodejs.dev/en/about/releases/).

Reply:
sounds good - looks like it is now `18.18.2`